### ISSUE TITLE
Deprecation warnings when using cask because of zipped

### ISSUE THREAD

**Issue by OndrejSpanel** [com-lihaoyi/cask#151]
This line from #137 is causing deprecation warnings when compiling against 0.10.1 with deprecations enabled:

https://github.com/com-lihaoyi/cask/blame/fae83f8ed17518574f0f6e1200bbd8a049bc1811/cask/src-3/cask/router/Macros.scala#L264

```scala
(sigss, argss, ctxs).zipped.map { case (sigs, args, ctx) =>
```

With fatal warnings this is fatal (sic):

> [error] C:\home\runner\work\cask\cask\cask\src-3\cask\router\Macros.scala: method zipped in class Ops is deprecated since 2.13.0: Use xs.lazyZip(ys).lazyZip(zs)

**Comment by jodersky** [comment]
mea culpa on this one. I think if you replace that line with `sigss.lazyZip(argss).lazyZip(ctxs)`, as the message suggests, the warning should go away.

**Comment by OndrejSpanel** [comment]
This is in a macro, therefore you do not get the warnings only when compiling the library, but also when you are just using it (linking to it).

**Comment by OndrejSpanel** [comment]
I will try to prepare a PR.

**Comment by jodersky** [comment]
Closed by #152
